sidebars.js

```diff
@@ -1,6 +1,30 @@
 const sidebars = {
-  docsSidebar: [{ type: "autogenerated", dirName: "docs" }],
-  referenceSidebar: [{ type: "autogenerated", dirName: "reference" }],
+  docsSidebar: [
+    { type: "autogenerated", dirName: "docs" },
+    {
+      type: "html",
+      value: "<hr/>",
+      className: "margin-vert--none",
+    },
+    {
+      type: "doc",
+      id: "reference/global-options",
+      label: "Reference",
+    },
+  ],
+  referenceSidebar: [
+    { type: "autogenerated", dirName: "reference" },
+    {
+      type: "html",
+      value: "<hr/>",
+      className: "margin-vert--none",
+    },
+    {
+      type: "doc",
+      id: "docs/introduction/how-does-pants-work",
```

Comment:
This doesn't need to be solved now, but just exploring the space: what happens if we want to change this entry page?

For instance, we add some awesome new landing page for Pants 2.34, that doesn't exist in 2.33 or earlier.

Is it possible to customise per version? Or use some generic entry point like `docs` that each version redirects itself?

Reply:
Yes, there is per-version sidebars (this particular file is for `docs/`, note all the other files in this PR (one per version)

Reply:
Oh, From the file names, I assumed they were auto-generated from this one... are they all manually maintained? When we add a new version do we have to manually copy it across?

Reply:
Nope, it's handled automatically by the versioning scripts. Check out the Docusaurus versioning docs

Reply:
Ah, I see https://docusaurus.io/docs/versioning#tagging-a-new-version

Once it's tagged, I assume they're then manually maintained? As in, you made these edits "manually" (by copying the file around, I imagine)?

Reply:
Yup! That's right